### ISSUE TITLE
stuck in node auth_and_test.js loop

### ISSUE THREAD

**Issue by ice22dragon** [hermanho/MMM-GooglePhotos#22]
This is probably a stupid user error but I've installed Magic Mirror and followed the MMM-GooglePhotos up through the node auth_and_test.js. When i run this, the browser opens like its supposed to and I select the account I am wanting to use. But then i get an error page saying "This app isn't verified" I tried going through the "advance" link to continue on to Magic Mirror but it just takes me back to the page where i select which account to use. I can't get past this point. Please help.
![AppVerificationError](https://user-images.githubusercontent.com/52040164/59853300-30567400-9336-11e9-8486-482685d970af.jpg)

**Comment by eouia** [comment]
every google app should be verified before released publicly.(e.g. commercial usage, registration in app store, and for some purposes)
But our module is not for anonymous people, it is just for you. non-commercial.
so we don't need to verification. this module will be always dev-stage.

**Comment by ice22dragon** [comment]
So then why is it asking for verification? All I did was create a new project, name it, enable the Google Photo API and create credentials. I got the .json file just fine so I assumed I  did this correct...? I didn't add any urls or anything except the name when i did the credentials. What could I have done to make them want to verify the app?

**Comment by ShakirAkbari** [comment]
I had trouble with installing correctly too. try creating a new google account and starting from scratch. 

eouia helped me troubleshoot in my issue - maybe try the steps we did here: https://github.com/eouia/MMM-GooglePhotos/issues/11

**Comment by ice22dragon** [comment]
I've tried starting from scratch, both with the install and with setting up the Google API credentials. I still can't get mine to work.

**Comment by eouia** [comment]
Verification is not needed. Go to console.developers.google.com then select your project. `API` > `Credentials`, You can find your oAuth Consent screen. Just set `Application Name` and `supporter email` and leave others empty or blank. And just `Save` (NOT `Submit and verification`).
If you are failed again, Remove your Google Project itself then create new one.

**Comment by ice22dragon** [comment]
I have tried removing and creating a new project. Same problem. Today I found this...
https://developers.google.com/photos/library/guides/authentication-authorization

It said that if your application accesses Google Photo APIs then it requires verification. I'm guessing they changed something on their end... 

![GooglePhotoRequiresVerification](https://user-images.githubusercontent.com/52040164/61084784-66968900-a3f4-11e9-9090-29093d62d6c8.JPG)

**Comment by ice22dragon** [comment]
I can't even figure out how to force it to go through the verification review process, the button is always grayed out. Just out of curiosity, do i have to have one of those Google Cloud Platform accounts for this to work? I really don't know what I'm doing wrong. As far as I can tell, I've done everything just as yall have said.

**Comment by ice22dragon** [comment]
Ok. I FINALLY got the node auth_and_test.js to run and this is what I did...

When the auth and test opened the browser window, instead of clicking on my account, i copied the url it directed me to, emailed it to myself and then opened that url on my computer rather than the PI. For whatever reason, that broke the loop and finally gave me the auth code, which i turned around and emailed back to myself on the Pi to finish the install. 

Still not sure what the issue was but I got it working. Thanks for having patience.
